src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -164,15 +164,15 @@ private long getLastCommitMillis() {
     }
 
     private void shutdownGracefully(final String reason) {
-        final Scope shutDownScope = TracingService.activateSpan(getContext().getCurrentSpan(), false);
-        TracingService.logStreamCloseReason(shutDownScope, "Shutting down gracefully. Reason: " + reason);
+        TracingService.logStreamCloseReason(getContext().getCurrentSpan(),
+                "Shutting down gracefully. Reason: " + reason);
         getLog().info("Shutting down gracefully. Reason: {}", reason);
         switchState(new ClosingState(this::getUncommittedOffsets, this::getLastCommitMillis));
     }
 
     private void pollDataFromKafka() {
         if (eventConsumer == null) {
-            TracingService.logErrorInSpan(TracingService.activateSpan(getContext().getCurrentSpan(), false),
+            TracingService.logErrorInSpan(getContext().getCurrentSpan(),
```

Comment:
user will definitely not need it, it is quite internal exception, which we could track in the logs